### DISCUSSION TITLE
AdGuard Home Docker Container - how to get it to stop making everything with root:root permissions?

### DISCUSSION THREAD

**Discussion by SawkeeReemo** [AdguardTeam/AdGuardHome#6982] [Q&A]
My subject says it all. For some reason the AdGuard docker container makes all of its files with root:root permissions which breaks it completely for me. Outside of me having to always fix all the permissions, how do you get it to stop doing that?

**Comment by SawkeeReemo** [comment]
Bueller?

**Comment by ammnt** [comment]
I have the same issue. I just want to run my container with unprivileged user in Docker Compose:
```
services:
  dns:
    user: "1000:1000"

```
But it is not possible because everything still works with root permissions. Even if I set my user in config file:
```
os:
  group: "1000"
  user: "1000"
```
These errors I have when I try to figure out:
`
2025/09/20 19:06:26.146063 [error] parsing configuration file: open /opt/adguardhome/conf/AdGuardHome.yaml: permission denied
`
or
`
2025/09/20 19:08:47.603453 1#1 [fatal] setting group: looking up group: group: unknown group 1000
`
or
`
2025/09/20 19:10:46.895663 1#1 [fatal] setting group: setting gid: operation not permitted
`
and much more. @ainar-g, please take a look. Maybe we can do this somehow, but I don't know how exactly🤔

Thanks!
